.github/workflows/iroha2-dev-pr-wasm.yaml

```
@@ -23,8 +27,11 @@ env:
   RUSTUP_TOOLCHAIN: nightly-2022-12-22
 
 jobs:
-  static-analysis:
+  wasm-static-analysis:
```

```suggestion
 static-analysis:
```

it's obvious from the workflow name